src/myframe.h

```diff
@@ -555,21 +555,22 @@ struct MyFrame : wxFrame {
         }
 
         wxMenu *markmenu = new wxMenu();
-        MyAppend(markmenu, A_MARKDATA, _(L"&Data"));
-        MyAppend(markmenu, A_MARKCODE, _(L"&Operation"));
-        MyAppend(markmenu, A_MARKVARD, _(L"Variable &Assign"));
-        MyAppend(markmenu, A_MARKVARU, _(L"Variable &Read"));
-        MyAppend(markmenu, A_MARKVIEWH, _(L"&Horizontal View"));
-        MyAppend(markmenu, A_MARKVIEWV, _(L"&Vertical View"));
+        MyAppend(markmenu, A_MARKDATA, _(L"&Data\tALT+d"));

```

Comment:
do these not clash with the menus or anything else?

Reply:
I did check this but I revisited your concern. RegEx `\\talt\+[ador.,]"` on the whole project returns 

1. myframe.h:563 MyAppend(markmenu, A_MARKVIEWV, _(L"&Vertical View\tALT+,"));
2. myframe.h:558MyAppend(markmenu, A_MARKDATA, _(L"&Data\tALT+d"));
3. myframe.h:559MyAppend(markmenu, A_MARKCODE, _(L"&Operation\tALT+o"));
4. myframe.h:560MyAppend(markmenu, A_MARKVARD, _(L"Variable &Assign\tALT+a"));
5. myframe.h:561MyAppend(markmenu, A_MARKVARU, _(L"Variable &Read\tALT+r"));
6. myframe.h:562MyAppend(markmenu, A_MARKVIEWH, _(L"&Horizontal View\tALT+."));

There should be no collision.

Reply:
I'm talking about being able to open the menus using the keyboard. For example, currently pressing ALT+e brings up the edit menu, and then ALT+d selects `Delete After` from the menu. Your shortcuts also have a ALT+d. Can you check that the above sequence still works?

Or more directly, if we ever added a Debug menu, we couldn't reach that with ALT+d. The ALT+LOWERCASE space is shared between menus and this specific menu.

If they indeed can clash, we might be better off with CTRL+ALT+.. for these?

Reply:
Oh I see. I rarely use this menubar navigation - thanks! In case of possible l10n there is no reliable way to avoid all ALT+? methods. I changed to CTRL+ALT+? pattern.

Reply:
I don't use it either.. but some might :)